docs/pages/_app.js

```diff
@@ -315,6 +317,11 @@ function AppWrapper(props) {
         ))}
         <meta name="mui:productId" content={productId} />
         <meta name="mui:productCategoryId" content={productCategoryId} />
+        <style>{`
+button[data-ga-event-action="stackblitz"] {
+  display: none;
+}
+`}</style>
```

Comment:
Stackblitz demos do not work: https://stackblitz.com/edit/react-ajjlj7?file=Demo.tsx

We can hide the Stackblitz buttons if we want to proceed with it.

Reply:
A similar issue was reported in https://github.com/stackblitz/core/issues/3194